index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       'silvermine/no-multiline-var-declarations': 'error',
       'silvermine/indent': [ 'error', 3, { 'VariableDeclaratorOffset': { 'var': 1, 'let': 1, 'const': 3 }, 'SwitchCase': 1 } ],
 
-      'comma-dangle': [ 'error', 'only-multiline' ],
+      'comma-dangle': [ 'error', 'always-multiline' ],
```

Comment:
So if the trailing comma causes an error in browsers, shouldn't we do `'comma-dangle': [ 'error', 'never' ]` here and then do the `always-multiline` in our node config?

It seems like we're going to be asking for errors otherwise, especially since those working on both code bases will have to remember to switch back and forth.